pkg/tests/tasks/migration/migration_test.go

```diff
@@ -147,6 +147,148 @@ spec:
 	})
 }
 
+func TestMigrationSimpleMultiTenant(t *testing.T) {
+	test.NewTest(t).MinVersion(version.SMCP_2_6).Groups(test.Migration).Run(func(t test.TestHelper) {
+		// delete mesh namespace from previous tests
+		t.LogStep("Cleanup any lingering namespaces from previous runs")
+		oc.DeleteTestBoundNamespaces(t)
+
+		t.Cleanup(func() {
+			oc.DeleteTestBoundNamespaces(t)
+			// clean up bookinfo
+			oc.DeleteFile(t, ns.Bookinfo, migrationGateway)
+			app.Uninstall(t, app.Bookinfo(ns.Bookinfo))
+		})
+
+		t.LogStep("Install SMCP tenant-a 2.6 in multitenant mode")
+
+		smcpA := ossm.DefaultSMCP()
```

Comment:
This is now failing after this change - https://github.com/maistra/maistra-test-tool/commit/da12cde47fc2837b32fc732f52a1a6acdb638de8#diff-0e54e2867b5c8d47a5bd9ae361e741b091a106765bbe10a4928aafa24baf7b1aR85